diff --git a/node_modules/nuxt-open-fetch/dist/runtime/plugin.mjs b/node_modules/nuxt-open-fetch/dist/runtime/plugin.mjs
index 686c6d6..20313f7 100644
--- a/node_modules/nuxt-open-fetch/dist/runtime/plugin.mjs
+++ b/node_modules/nuxt-open-fetch/dist/runtime/plugin.mjs
@@ -1,4 +1,5 @@
 import { defineNuxtPlugin, useRuntimeConfig } from "#app";
+import { createOpenFetch } from "./clients";
 export default defineNuxtPlugin(() => {
   const clients = useRuntimeConfig().public.openFetch;
   return {
